### DISCUSSION TITLE
AppleII-VGA problem with AppleII+ Clone and a Videx card

### DISCUSSION THREAD

**Discussion by mkrumov** [markadev/AppleII-VGA#52] [General]
Hi,
i have  AppleII+ Clone and a Videx card and try to start 80 columns mode. Videx VideoTerm card is in slot 3.I have enabled 80 column VGA support using the Configuration Disk image. But when I type pr#3 the Apple 'hangs' staying in the normal lores mode. I connect the output of the 80 columns card to the monitor and see the 80 columns.
And there is the 80 columns specfic steps?

**Comment by mkrumov** [comment]
The problem is solved by changing the firmware of the Videx VideoTerm card.
Regards!

**Reply by markadev** [reply]
OK thanks, glad you got it working!

The mode switching in the VGA card is controlled by the the same logic used by the Videx soft video switch (connected to one of the Apple II annunciator outputs).  Perhaps this wasn't supported by the original Videx firmware you were using.
